Project.toml

```diff
@@ -15,6 +15,7 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 
 [compat]
+Distances = "0.8"
```

Comment:
Freezing on Distances can be  constraining, since it is only for tests we can leave it unconstrained for the moment. If at one point it breaks we will see it on Travis and will re-add the constraint here